en/editor/publish/publish-xiaomi-quick-game.md

```diff
@@ -138,14 +136,14 @@ Debugging can currently be initiated from the command line.
 
 Subpackage rpk can be used according to your needs.
 
-Subpackage loading, that is, splitting the game content into several packages according to certain rules, only downloading the necessary packages when starting up for the first time. This necessary package is called **main package**. And the developer can trigger in the main package to download other sub-packages, which can effectively reduce the time spent on the first boot. To use this function, you need to set [Bundle Configuration](../../asset/bundle.md) in __Cocos Creator__, and the package will be automatically subpackaged when the setting is completed.
+Subpackage loading, that is, splitting the game content into several packages according to certain rules, only downloading the necessary packages when starting up for the first time. This necessary package is called **main package**. And the developer can trigger in the main package to download other sub-packages, which can effectively reduce the time spent on the first boot.
 
-> **Note**: the size of a single subpackage/main package must not exceed **5M**, and the sum of all packages must not exceed **10M**.
+To use this function, you need to set [Bundle Configuration](subpackage.md) in __Cocos Creator__, and the package will be automatically subpackaged when the setting is completed.
 
-When the build is complete, an `.rpk` file is generated in the `/build/xiaomi-quick-game/dist` directory.
+When the build is complete, an `.rpk` file is generated in the `build/xiaomi-quick-game/dist` directory.
 
-## Reference documentation
+The package size limit for single subpackage/main package, see [Xiaomi Quick Game Subpackage Rules [cn]](https://forum.cocos.org/t/topic/81887) for details.
```

Comment:
The package size limit for single subpackage/main package, see [Xiaomi Quick Game Subpackage Rules [cn]](https://forum.cocos.org/t/topic/81887) for details.
-->
The package size limitations for single subpackage/main package, please refer to the [Xiaomi Quick Game Subpackage Rules [cn]](https://forum.cocos.org/t/topic/81887) documentation for details.